sdkutil/event.go

```diff
@@ -1,17 +1,30 @@
 package sdkutil
 
 import (
-	"errors"
+	"path"
+	"path/filepath"
+	"reflect"
 	"strconv"
+	"strings"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/pkg/errors"
 )
 
 const (
 	akashEventMessageV1 = "akash.v1"
 
 	// EventTypeMessage defines the Akash message string
 	EventTypeMessage = akashEventMessageV1
+
+	akashHexLen    = 20
+	akashBech32Len = 44
```

Comment:
are any components of these defined in sdk or tm?

Reply:
`akashBech32Len` is custom due to `akash` prefix to the string, the AddrLen can be pulled from `sdk`. I've updated the const to match sdk value, could just replace it.